src/app/contact/contact.component.html

```diff
@@ -0,0 +1,34 @@
+	<div class="figure">
+		<img src = "../assets/img/contact.png" class = "img-fluid" style=" display:block; width:80%; margin:auto; margin-top: 5px; margin-bottom: 60px;">
+	</div>
+	<br>
+	<div class = " container box logos mx-auto d-block">
+		<div class = "row">
+			<div class = "col-md-4">
+				<a href = "https://systers-opensource.slack.com/"> <button type="button" class="btn btn-rounded btn-lg btn-block waves-light brand-c" mdbWavesEffect style="background-color: #54BCEB; width:50%; margin: auto;">Slack</button></a>
+			</div>
+			<div class = "col-md-4">
+				<a href="https://github.com/systers/communities/"><button type="button"  class="btn btn-rounded waves-light brand-c btn-lg btn-block" mdbWavesEffect style="background-color: #54BCEB;width:50%;margin: auto;" >Github</button></a>
+			</div>
+			<div class = "col-md-4">
+				<a href="http://systers.org/mailman/listinfo/systers-opensource"><button type="button" class="btn btn-rounded waves-light brand-c btn-lg btn-block" mdbWavesEffect style="background-color: #54BCEB;width:50%; margin: auto;">Mailing List</button></a>
+			</div>
+		</div>
+	</div>
+
+	<br><br><br>
```

Comment:
br tags is not good practice

Reply:
Resolved by improving on the code modularity. Also aligned all the contact numbers :)